tests/test_safeFileIo.py

```diff
@@ -147,14 +147,14 @@ def testNoChange(self):
             self.assertEqual(f.read(), "some test string")
 
 
-class TestOneThousandWriters(unittest.TestCase):
+class TestMultipleWriters(unittest.TestCase):
     """Test for efficient file updating with shared & exclusive locks by serializing a RepositoryCfg to a
     location 1000 times. When this was tested on a 2.8 GHz Intel Core i7 macbook pro it took about 1.3 seconds
```

Comment:
This is no longer 1000 times. Please update the docstring if anything else needs to change, too.